src/NBitcoin/ChainIndexer.cs

```diff
@@ -179,9 +185,9 @@ public virtual IEnumerable<ChainedHeader> EnumerateAfter(ChainedHeader block)
         }
 
         /// <summary>
-        /// TODO: Make this internal when the component moves to Stratis.Bitcoin
```

Comment:
Hmmm I remember this one..,. the idea was to move the component to `Stratis.Bitcoin` but we never did. So I'm not sure this PR is of any help? Pehaps just delete the TODO? Unless we have reason to move the ChainIndexer but then that willl be a big change.